app/views/welcome/index.html.erb

```diff
@@ -1,13 +1,23 @@
 <h1>Welcome</h1>
 
-<h2> Students </h2>
-<%= link_to 'All students', controller: 'students' %>
-<%= link_to 'New student', new_student_path %>
+<% if current_user.tutor_level? %>
+  <h2> Students </h2>
+  <% if current_user.tutor? %>
+    <%= link_to 'My students', tutor_path(1) %>
```

Comment:
Here, since we know the user is a tutor we can just grab the `tutor` where the `user_id == current_user.id`